### DISCUSSION TITLE
메인화면에서 글 목차가 대빵만하게 보여져서 문제입니다.

### DISCUSSION THREAD

**Discussion by MUNJB** [pronist/hELLO#106] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: windows11
- 브라우저: chrome
- 버전: 102.0.5005.115(공식 빌드) (64비트)
- 스킨 버전 (스킨 버전은 index.xml 에서 확인): 1.0

**설명**
작성자님처럼 메인 페이지가 '글-그림'이런 식으로 깔끔하게 나열되지 않고
그림이 엄청나게 크게 화면에 떡하니 있고 글이 그 밑에 나열되어 목록을 볼 수 없는 상태로 나옵니다.

**발생 과정**
https://pronist.dev/5#%EB%-B%A-%EC%-A%B-%EB%A-%-C%EB%--%-C
해당 글을 읽으며 스킨적용을 했습니다. 그랬더니 아래 스크린샷 처럼 나옵니다.

**스크린샷 및 링크**
https://monstock.tistory.com/
메인 페이지가 다음과 같이 뜹니다.
![캡처](https://user-images.githubusercontent.com/107293689/173173802-e5a6a5c5-ec7a-45ee-8261-e8c1ca48bbd5.JPG)

#60 문제와 비슷하건지 모르겠지만 해결책이 있을지요 ㅜㅜ

**Comment by pronist** [comment, marked as answer]
@MUNJB  안녕하세요!

스킨옵션에 리스트 스타일 변경하는 옵션이 있을겁니다. 그거 사용해서 변경하시면 됩니다. :)
